src/client/components/NewsletterCard.tsx

```diff
@@ -108,51 +106,134 @@ const subtitleDiv = css`
   }
 `;
 
+const spaceBottom = css`
+  margin-bottom: ${space[5]}px;
+  padding-bottom: ${space[1]}px;
+`;
+
 // @TODO: hacked background colour, should be fixed in future source checkbox implementation
 const checkBoxBackgroundColorBugFix = css`
   background: ${neutral[100]};
   z-index: 0 !important;
 `;
 
-const clockSVG = (
-  <svg
-    css={{ fill: brand[400] }}
-    width="15px"
-    height="15px"
-    viewBox="0 0 11 11"
-  >
-    <path d="M5.4 0C2.4 0 0 2.4 0 5.4s2.4 5.4 5.4 5.4 5.4-2.4 5.4-5.4S8.4 0 5.4 0zm3 6.8H4.7V1.7h.7L6 5.4l2.4.6v.8z" />
-  </svg>
-);
+export const iconStyles = css`
+  width: fit-content;
+  border-radius: 100%;
+  padding: 2px;
+  svg {
+    display: block;
+    fill: ${neutral[100]};
+    height: 22px;
+    width: 22px;
+  }
+  margin-bottom: ${space[1]}px;
+  margin-right: ${space[1]}px;
+
+  ${from.tablet} {
+    margin-top: ${space[2]}px;
+  }
+`;
+
+const descriptionBlock = css`
+  ${from.tablet} {
+    display: flex;
+    border-top: 1px solid ${neutral[86]};
+    border-bottom: 1px solid ${neutral[86]};
+  }
+`;
+
+const checkboxGroup = css`
+  ${from.tablet} {
+    // position: absolute;
+    // bottom: 0;
+  }
+`;
+
+const infoGroup = css`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+
+  ${from.tablet} {
+    padding-left: ${space[5]}px;
+  }
+`;
+
+const frequencyStyles = css`
+  display: flex;
+  align-items: center;
+  h2 {
+    margin-left: ${space[2]};
+  }
+`;
+
+const idColor: (arg0: string) => string | undefined = (id) => {
+  console.log(id);
+  if (id === 'today-us') {
+    return news[400];
+  }
+  if (id === 'the-long-read') {
+    return brand[400];
+  }
+  if (id === 'green-light') {
+    return news[400];
+  }
+  if (id === 'bookmarks') {
+    return culture[400];
+  }
+};
```

Comment:
Had another thought. We already have the enum `Newsletters` which can be imported from `'@/shared/model/Newsletter'`, would be preferable to use those ids instead of the `nameId` just so we don't end up with two different approaches.

Also noticed you are only handling `today-us`. For other locales (AU, GB, ROW) a colour won't be defined.

Reply:
Pushed an approach on my rebased branch that you might find useful? https://github.com/guardian/gateway/commit/de91169373fcbe28e6dbef8b809bd2c6ad89b8a1

Reply:
Very useful @liamness. Thank you!

Reply:
Hey as you mentioned you were waiting for reviews at standup, I thought it would be worth asking, have you looked into this yet? Looks like there will still be issues in non-US locales.